pkg/backup/restore/restore_manager.go

```diff
@@ -1120,12 +980,18 @@ func (rm *restoreManager) makeRestoreJob(restore *v1alpha1.Restore) (*batchv1.Jo
 	}
 
 	jobLabels := util.CombineStringMap(label.NewRestore().Instance(restore.GetInstanceName()).RestoreJob().Restore(name), restore.Labels)
+	// Add component label for prune job
+	if isPruneJob {
+		jobLabels["app.kubernetes.io/component"] = "prune"
```

Comment:
I think we don't need to add "prune" component. It should belong to restore component.